README.md

```diff
@@ -79,25 +79,3 @@ func newConnection(conn openvpn.IncomingConn) {
 
 For more usage information on both modes, see
 [the reference documentation](https://godoc.org/github.com/NordSecurity/gopenvpn/openvpn).
-
```

Comment:
Please add a reference to the license file here as well.

Reply:
Added.